core/modules/image/image.field.inc

```diff
@@ -695,6 +709,28 @@ function image_field_formatter_settings_form($field, $instance, $view_mode, $for
   $element['image_load']['auto']['#description'] = t('Each browser will determine whether or not to lazily load the image.');
   $element['image_load']['eager']['#description'] = t('Load this image immediately, regardless of where it is located on the page.');
 
+  $element['image_svg_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('SVG Images dimensions (attributes)'),
+    '#tree' => TRUE,
+  );
+
+  $element['image_svg_settings']['width'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Width'),
+    '#size' => 10,
+    '#field_suffix' => 'px',
```

Comment:
In the image field settings form, we are using the full word "pixels":
![image](https://github.com/backdrop/backdrop/assets/2423362/a8599a40-4a99-41bb-89be-cc2231310ea9)

Reply:
Good idea, done.